assets/testing_instructions.MD

```diff
@@ -0,0 +1,12 @@
+# Testing Instructions
+
+1) download the google assistant or Alexa app on your phone/tablet
+2) login with the email and password provided below
+
+username: cronpatientemail@gmail.com
+password: 50f407f21003
+
+3) in the main/home tab:
+     tap on the microphone for google 
+     tap on alexa icon for alexa
+4) say "hey google talk to clinic attendant" or "alexa open clinic attendant"
```

Comment:
Say "Hey Google, talk to Clinic Attendant" or "Alexa, open Clinic Attendant"

Reply:
that's all ,right?